### PR TITLE
[PickerInput]: added ability to pass rawProps to modal window

### PR DESCRIPTION
**Pull request by blackymax**

### CODE REVIEW

File: epam-promo/components/pickers/PickerModal.tsx [hunk 1]
```diff
@@ -117,8 +117,8 @@ export class PickerModalImpl<TItem, TId> extends PickerModalBase<TItem, TId> {
 export class PickerModal<TItem, TId> extends React.Component<PickerModalProps<TItem, TId>> {
     state = { selection: this.props.initialValue };
     lens = Lens.onState<any>(this);
-
     render() {
```
Comment (AlekseyManetov) [inline epam-promo/components/pickers/PickerModal.tsx:120]: remove console.log

Comment (blackymax) [reply epam-promo/components/pickers/PickerModal.tsx:120]: done.

File: uui-core/types/components/PickerBase.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { ReactNode } from "react";
-import { IAnalyticableOnChange, IEditable } from "../props";
+import { IAnalyticableOnChange, IEditable, IHasRawProps } from "../props";
```
Comment (AlekseyManetov) [inline uui-core/types/components/PickerBase.ts:2]: remove unused IHasRawProps import

**Review by AlekseyManetov — CHANGES_REQUESTED** [review]

File: uui-core/types/components/PickerBase.ts (hunk 2 above)
Comment (blackymax) [reply uui-core/types/components/PickerBase.ts]: done.

**Pull request merged**